src/content/docs/docs/protocol-reference/transactions/pseudo-transaction-types/cron.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Cron
+description: >-
+  The Cron pseudo-transaction is automatically inserted by the Cron engine when
+  a scheduled Hook execution is due.
+---
+\[[Source](https://github.com/Xahau/xahaud/blob/dev/src/ripple/app/tx/impl/Cron.cpp)]
+
+_(Added by the [CronSet amendment][].)_
+
+The `Cron` pseudo-transaction is automatically inserted into the ledger by the Cron engine when a scheduled Hook execution is due. This pseudo-transaction triggers the execution of the Hook on the account specified in the `Owner` field.
+
+### Example
+
+```json
+{
+  "TransactionType": "Cron",
+  "Account": "0000000000000000000000000000000000000000",
+  "Fee": "0",
+  "Sequence": 0,
+  "SigningPubKey": "",
+  "TxnSignature": "",
+  "Owner": "rWYkbWkCeg8dP6rXALnjgZSjjLyih5NXm"
+}
```

Comment:
need `LedgerSequence` field here, and below (Fields) (same as other pseudo txn's one)